### DISCUSSION TITLE
NameError: name 'init' is not defined. Did you mean: 'int'?

### DISCUSSION THREAD

**Discussion by julianogimenez** [new92/igfi#20] [General]
Do you know why Im getting that message:
NameError: name 'init' is not defined. Did you mean: 'int'?
?

**Comment by new92** [comment]
Hey there, ok so I suppose that you used the version 3 of the script (mainV3.py). There was a bug in the installation process which was causing this error. I've made some changes so now it shouldn't display this error. Please uninstall the script manually and repeat the installation process. It should work fine now. If you have any other troubles feel free to contact me again anytime !

**Reply by Mohamudhussein** [reply]
Hi there, mine is in the pip install, it says command not found

**Reply by new92** [reply]
Hey, in which part does the `command not found` error appear ? Like do you see any message before the error message ?

**Comment by new92** [comment]
@julianogimenez @sadamdicker have you been able to resolve the issues?
